portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +36,58 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  /** Generates a random quote from the ArrayList Quotes and converts the quote to a JSON object.*/
```

Comment:
`Quotes` --> `Questions`?

Reply:
actually it's not even a random quote anymore. Just say something like it returns all the questions posted.